wdn/templates_4.0/examples/notice.html

```diff
@@ -41,28 +38,31 @@
         <a href="#" title="Close this notice">Close this notice</a>
     </div>
     <div class="message">
-        <h4>Important Alert!</h4>
+        <p class="title">Important Alert!</p>
         <p>This is my content. <a href="#">This is some link text</a></p>
     </div>
 </div>
-<div class="wdn_notice overlay-header duration-10">
+<div class="wdn_notice" data-overlay="header" data-duration="10">
     <div class="close">
         <a href="#" title="Close this notice">Close this notice</a>
     </div>
     <div class="message">
-        <h4>I'm Floating!</h4>
-        <p>This notice has classes <em>overlay-header</em> and <em>duration-10</em> and will vanish on its own.</p>
+        <p class="title">I'm Floating!</p>
+        <p>This notice overlays the header and will vanish on its own in 10 seconds.</p>
     </div>
 </div>
-<div class="wdn_notice affirm overlay-maincontent duration-15">
+<div class="wdn_notice affirm" data-overlay="maincontent" data-duration="15">
     <div class="close">
         <a href="#" title="Close this notice">Close this notice</a>
     </div>
     <div class="message">
-        <h4>Success!</h4>
-        <p>This notice has classes <em>affirm</em>, <em>overlay-maincontent</em>, and <em>duration-15</em></p>
+        <p class="title">Success!</p>
+        <p>This notice overlays the maincontent and will vanish on its own in 15 seconds.</p>
     </div>
 </div>
+<script type="text/javascript">
+WDN.initializePlugin('notice');
+</script>
 </div>
 </body>
 </html>
```

Comment:
I'm not trying to be confrontational either.  I'd argue that feature creep is not this.  Pull requests should be reviewed for code style, bugs, usability and risk.  Risk and usability includes accessibility.  If care was taken to refactor something to make it better (and this is much better than it was before), accessibility should be reviewed in the pull request.

Feature creep on the other hand is adding additional superficial (maybe the wrong word) features, such as different fading transitions, or different colors because they would look nice, or a different widget entirely.

When a widget is updated, all aspects of that widget in terms of code and usability should be visited and addressed.

Perhaps that is just my professional opinion.